src/field/captcha/recaptchav2.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Set } from 'immutable';
+import * as l from '../../core/index';
```

Comment:
again, use of a non-descriptive variable `l` makes this harder to understand. perhaps `Lock`?

Reply:
as per https://github.com/auth0/lock/pull/1845#discussion_r410147850, this is library convention and changing it is outside the scope of this PR.